src/tools/dev/scripts/bv_support/bv_mpich.sh

```diff
@@ -134,6 +139,7 @@ function build_mpich
                   CXXFLAGS="$MPICH_CXXFLAGS $MPICH_CXX_OPT_FLAGS"\
                   FFLAGS="$MPICH_FCFLAGS"\
                   ./configure ${mpich_opts} \
+                  LDFLAGS="${ldflags}" \
```

Comment:
@biagas can you take a quick look here. Do you think an empty LDFLAGS on the configure command could cause issues anywhere?

```
./configure LDFLAGS=

Reply:
Not sure. Could you make the whole thing a temp var, so that it would possibly pass an empty string. Something like:
```
mpich_ldflags=""
if [[ "ldflags" != "" ]]; then
    mpich_ldflags="LDFLAGS=${ldflags}"
fi
```